src/handycon/handhelds/aya_gen2.py

```diff
@@ -49,10 +49,10 @@ async def process_event(seed_event, active_keys):
     # BUTTON 5 (Default: MODE) Big button
     if active_keys in [[96, 105, 133], [88, 97, 125]] and button_on == 1 and button5 not in handycon.event_queue:
         handycon.event_queue.append(button5)
-        await handycon.emit_now(seed_event, button2, 1)
-    elif active == [] and seed_event.code in [88, 96, 97, 105, 125, 133] and button_on == 0 and button5 in handycon.event_queue:
-        handycon.event_queue.remove(button5)
         await handycon.emit_now(seed_event, button5, 1)
+    elif active_keys == [] and seed_event.code in [88, 96, 97, 105, 125, 133] and button_on == 0 and button5 in handycon.event_queue:
```

Comment:
Good catch on this, but I'd prefer it be in a separate PR as its unrelated to the main issue.